client/src/lib/TinyMCEActionRegistrar.js

```diff
@@ -36,6 +36,17 @@ class TinyMCEActionRegistrar {
     const name = createIdentifier(configId, menu);
     const actions = this.getActions(menu, configId, true);
 
+    /* eslint-disable no-param-reassign */
+    action.type = 'menuitem';
+    // To reduce upgrade pain from tinymce4 to tinymce6 (cms4 to cms5) we've added this
+    // as this would mean any custom plugins using this class to register commands won't
+    // need to update the syntax for doing so
+    if (action.hasOwnProperty('onclick')) {
+      action.onAction = action.onclick;
+      delete action.onclick;
+    }
```

Comment:
I'm happy to remove this if there is any pushback at all - people will probably have to update other parts of their plugins anyway so this doesn't really help _all_ that much.